### PR TITLE
Enhance README.md

### LINKED ISSUE #22: Enhance README
Update the README with proper setup instructions and project description.

### PR DESCRIPTION
**Pull request by Akshu-on-github**
# Related Issue
- closes #22 

# Proposed Changes
- Enhanced the flow of the document
- Tidied up the links

# Additional Info
- Made a few changes to the instructions to improve readability

# Checklist
(None applicable)
- [ ] Tests
- [ ] Translations
- [ ] Documentation

# Screenshots

Original              |  Updated
:--------------------------:|:-----------------------:
** original screenshot **
image
   |  ** updated screenshot **

### CODE REVIEW

**Comment by adityabisoi** [comment]
@AkshayaKulasekaran You should not make a PR unless you're assigned to an issue.

File: README.md [hunk 1]
```diff
@@ -2,20 +2,19 @@
...
+You can refer to the documentation [here](https://bit.ly/3kRkM2e)
 
-## 🏆 Contributing
-
-Please read  [CONTRIBUTING.md](CONTRIBUTING.md)  for information on how to contribute to github-rest-api-v3.
+## Contributing 🏆
+Please read [CONTRIBUTING.md](CONTRIBUTING.md) before contributing to the repository (github-rest-api-v3)
 
 ## Support
 <a href="https://gitter.im/github-rest-api-v3/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge" rel="some text">![Foo](https://icon-icons.com/icons2/2530/PNG/72/gitter_button_icon_151850.png
```
Comment (adityabisoi) [inline README.md:18]: Add an `Open Source Events` section mentioning the project being a part of NJHACK. Add NJHACK logo as well.

**Comment by Akshu-on-github** [comment]
> @AkshayaKulasekaran You should not make a PR unless you're assigned to an issue.

Sorry about that!

File: README.md (hunk 1 above)
Comment (Akshu-on-github) [reply README.md:18]: @adityabisoi, I'm afraid I can't find anything about NJHACK, could you send me the relevant link(s)? I've added the outline of the subsection, but require further details to flesh it out

Comment (adityabisoi) [reply README.md:18]: Just add 'This project is a part of NJACK' with NJHACK pointing a link to the website https://njackwinterofcode.github.io/
And, add the logo
nwoc-logo

Comment (Akshu-on-github) [reply README.md:18]: How about now?

Comment (adityabisoi) [reply README.md:18]: Make it smaller, and centre it.
https://stackoverflow.com/questions/12090472/github-readme-md-center-image

Comment (Akshu-on-github) [reply README.md:18]: Done, I've set it at 200px

**Pull request merged**